content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst

```diff
@@ -133,507 +155,593 @@ types available by default:
+  POS, which is usually the same address as the Company. For example, if a Company has multiple
+  stores (fiscal locations) then AFIP will require the Company to have one AFIP POS per location.
+  This location will be printed in the invoice report.
+- :guilabel:`Unified Book`: when AFIP POS System is Preimpresa, then the Document Types (applicable
+  to the journal) with the same letter will share the same sequence. For example:
 
-- Invoice: FA-A 0001-00000002.
-- Credit Note: NC-A 0001-00000003.
-- Debit Note: ND-A 0001-00000004.
+  - Invoice: FA-A 0001-00000002.
+  - Credit Note: NC-A 0001-00000003.
+  - Debit Note: ND-A 0001-00000004.
 
 Sequences
 ~~~~~~~~~
-In case that you want to synchronize the next number in the sequence in Odoo based on the next
-number in the AFIP POS, the next button that is visible under :ref:`developer mode <developer-mode>`
-can be used:
 
-.. image:: argentina/argentina_edi_05.png
-   :align: center
+For the first invoice, Odoo syncs with AFIP automatically and brings the last sequence used.
 
 .. note::
-  When creating the Purchase journals, it's possible to define if they can be related to document 
-  types or not.  In case that the option to use documents is selected, there is no need to manually 
-  associate the document type sequences as the document number is provided by the vendor.
-
+   When creating :guilabel:`Purchase Journals`, it's possible to define whether they are related
+   to Document Types or not. In the case where the option to use documents is selected, there
+   would be no need to manually associate the Document Type sequences, since the document number is
+   provided by the vendor.
 
 Usage and testing
 =================
 
 Invoice
 -------
 
-After the partners and journals are created and configured, when the invoices are created the will
-have the next behaviour:
+The information below applies for invoice creation once the partners and journals are created and
+properly configured.
 
-Document type assignation
+Document Type assignation
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Once the partner is selected the document type will filled automatically, based on the AFIP document
-type:
-
-**Invoice for a customer IVA Responsable Inscripto, prefix A**.
+Once the partner is selected, the Document Type will be filled in automatically based on the AFIP
+Document Type:
 
-.. image:: argentina/argentina11.png
-   :align: center
+- **Invoice for a customer IVA Responsable Inscripto, prefix A**: type of document that is going to
+  show all the taxes in detail and customer's information.
 
-**Invoice for an end customer, prefix B**.
+  .. image:: argentina/prefix-a-invoice-for-customer.png
+     :alt: Invoice for a customer IVA Responsable Inscripto, prefix A.
 
-.. image:: argentina/argentina12.png
-   :align: center
+- **Invoice for an end customer, prefix B**: type of document that does not detail the taxes, since
+  the taxes are included in the total amount.
```

Comment:
Remove nested colon (`:`)
```suggestion
- **Invoice for an end customer, prefix B** is the type of document that does not detail the taxes,
  since the taxes are included in the total amount.
```